src/build/env.py

```diff
@@ -119,15 +131,10 @@ def install(self, requirements):  # type: (Iterable[str]) -> None
             req_file.write(os.linesep.join(requirements))
```

Comment:
A comment should explain why a requirements file is used and why `delete` is set to `False` and the file deleted manually on line 144 both of which are non-obvious.

Reply:
out of scope for this PR